rdtools/soiling.py

```diff
@@ -85,6 +87,11 @@ def _calc_daily_df(self, day_scale=14, clean_threshold='infer',
             The daily precipitation threshold for defining precipitation cleaning events.
             Units must be consistent with ``self.precipitation_daily``.
         '''
+        if (day_scale % 2 == 0) and ('shift' in clean_criterion):
+            warnings.warn('An even value of day_scale was passed. An odd value is '
+                          'recommended, otherwise, consecutive days may be erronsiously '
+                          'flagged as cleaning events. '
+                          'See https://github.com/NREL/rdtools/issues/189')
```

Comment:
Should we be specifying a warning type?  The default is UserWarning, which I think is appropriate, but wanted to bring it up for discussion.  See https://docs.python.org/3/library/warnings.html#warning-categories

Reply:
I agree the default `UserWarning` is appropriate. Do you think we should be explicit? Or OK to rely on default?

Reply:
Fine with me to use the default 👍 I was wondering if emitting the default warning is like raising the base `Exception` class instead of a more specific exception, but I think UserWarning is fine.